definitions/velocity.proto

```diff
@@ -42,3 +42,57 @@ message DetectMotionRequest {
   required PlatformType platform  = 6;
   required double timestamp       = 7;
 }
+
+message HandshakeRequest {
+  required string authToken       = 1;
```

Comment:
Can you check if it's easy to use ``Authorization`` header with SocketRocket..? At least on the server side it would be nice to extract this even before we start exchanging WebSocket messages.

Reply:
It is possible to set header on native code, but in browser seems it isn't working. But there are other options, e.g.  pass token as  part of URL or fetch WS URL and just then connect via WS. But for now I would like to keep it, maybe just make it optional ?

Reply:
``optional`` seems good